test/libsolidity/ASTPropertyTest.cpp

```diff
@@ -108,30 +108,30 @@ void ASTPropertyTest::readExpectations()
 	m_expectation = formatExpectations(false /* _obtainedResult */);
 }
 
-void ASTPropertyTest::extractTestsFromAST(Json::Value const& _astJson)
+void ASTPropertyTest::extractTestsFromAST(Json const& _astJson)
 {
-	std::queue<Json::Value> nodesToVisit;
+	std::queue<Json> nodesToVisit;
 	nodesToVisit.push(_astJson);
 
 	while (!nodesToVisit.empty())
 	{
-		Json::Value& node = nodesToVisit.front();
+		Json& node = nodesToVisit.front();
 
-		if (node.isArray())
+		if (node.is_array())
 			for (auto&& member: node)
 				nodesToVisit.push(member);
-		else if (node.isObject())
-			for (std::string const& memberName: node.getMemberNames())
+		else if (node.is_object())
+			for (auto const& [memberName, _]: node.items())
 			{
 				if (memberName != "documentation")
 				{
 					nodesToVisit.push(node[memberName]);
 					continue;
 				}
 
-				std::string nodeDocstring = node["documentation"].isObject() ?
-					node["documentation"]["text"].asString() :
-					node["documentation"].asString();
+				std::string nodeDocstring = node["documentation"].is_object() ?
```

Comment:
```suggestion
				std::string nodeDocstring = value.is_object() ?
```
It doesn't exactly look like it, since this is a rather weird loop - but this is also just ``value`` if you replace ``_`` by ``value`` - also below.